HandmadeMath.h

```diff
@@ -1465,6 +1516,124 @@ HMM_QuaternionV4(hmm_vec4 Vector)
     return(Result);    
 }
 
+HINLINE hmm_quaternion
+HMM_Quaternion_Addition(hmm_quaternion QuaternionOne, hmm_quaternion QuaternionTwo)
+{
+    hmm_quaternion Result = {0};
+
+    Result.X = QuaternionOne.X + QuaternionTwo.X;
+    Result.Y = QuaternionOne.Y + QuaternionTwo.Y;
+    Result.Z = QuaternionOne.Z + QuaternionTwo.Z;
+    Result.W = QuaternionOne.W + QuaternionTwo.W;
+
+    return(Result);
```

Comment:
Nitpick: I see `return (Result);` more often than `return(Result);` in the code. But we are pretty inconsistent, it seems. @StrangeZak, which do you prefer?